CHANGELOG/CHANGELOG-0.17.md

```diff
@@ -0,0 +1,44 @@
+# Release notes for v0.17.0
+
+# Changelog since v0.16.0
+
+## Changes by Kind
+
+### Uncategorized
+
+- Update kubernetes dependencies to v1.29.0 ([#156](https://github.com/kubernetes-csi/csi-lib-utils/pull/156), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
+- Update kubernetes dependencies to v1.29.0-rc.1 ([#155](https://github.com/kubernetes-csi/csi-lib-utils/pull/155), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
```

Comment:
this was an intermediate state that does not need  release note

Reply:
Done